packages/accounts/src/light-account/erc1271/utils/hashTypedData.ts

```diff
@@ -0,0 +1,278 @@
+// Copied from https://github.com/wevm/viem/blob/viem%402.9.12/src/utils/signature/hashTypedData.ts.
+// Unfortunately, viem does not export any of these functions
```

Comment:
wait yea they do, we use `hashTypedData` from `viem` in a few places. [eg `multi-owner/signer.ts`](https://github.com/alchemyplatform/aa-sdk/blob/main/packages/accounts/src/msca/plugins/multi-owner/signer.ts#L77)

Reply:
Oh you're right, I should update the comment. Viem exports `hashTypedData`, but it doesn't export `hashStruct` or `hashDomain` which are the ones we need.

Reply:
ahhhh gotcha